x/auth/tx/config.go

```diff
@@ -189,17 +189,16 @@ func NewTxConfigWithOptions(protoCodec codec.Codec, configOptions ConfigOptions)
 	var err error
 	opts := &configOptions
```

Comment:
curious, why the reassignment to `opts` here?

Reply:
It does look odd, probably a relic of an old code path. I've simplified it.